src/pages/HomePage.jsx

```diff
@@ -9,21 +14,56 @@ export default function CHRAT({messages, onNewMessage}) {
   }
   return (
     <>
-      <ChratBody>
-        <Header>
+      <StyledChratDiv>
+        {/* Header */}
+        <StyledHeader>
           <h1>chRat</h1>
-        </Header>
-        <Main id={'mainChat'}>
-          <Chat messages={messages} goToScrollDown={goToScrollDown} />
-          {/* {window.scrollTo(0, document.body.scrollHeight)} */}
-        </Main>
-        <Input addNewMessage={onNewMessage} />
-      </ChratBody>
+        </StyledHeader>
+        <Routes>
+          {/* Chat Inhalt Page */}
+          <Route
+            path="/"
+            element={
+              <>
+                {/* Chat Inhalt */}
+                {/* <StyledMain id={'mainChat'}> */}
+                <StyledMain>
+                  <Chat messages={messages} goToScrollDown={goToScrollDown} />
+                </StyledMain>
+                {/* Message Input */}
+                <Input onNewMessage={onNewMessage} />
+              </>
+            }
+          />
+          {/* About Page Route */}
+          <Route
+            path="/about"
+            element={
+              <>
+                <About />
+              </>
+            }
+          />
+          {/* Imprint Page Route */}
+          <Route
+            path="/imprint"
+            element={
+              <>
+                <Imprint />
+              </>
+            }
+          />
+        </Routes>
+        {/* Hamburger Menü */}
+        <Nav />
+      </StyledChratDiv>
     </>
   );
 }
 
-const ChratBody = styled.div`
+/* {window.scrollTo(0, document.body.scrollHeight)} */
```

Comment:
Please remove unused code 😉

Reply:
jah ok where i can leave unncomment code ? i would use these later maybe so.. ?

Reply:
If you want to use code later, you can always find them in your git history :)